vulnfeeds/cpp/main.go

```diff
@@ -181,6 +183,18 @@ func CVEToOSV(CVE cves.CVEItem, repos []string, repobasedir string, directory st
 		return fmt.Errorf("Failed to write %s: %v", outputFile, err)
 	}
 	Logger.Infof("Generated OSV record from %s for %q", CVE.CVE.CVEDataMeta.ID, CPE.Product)
+	if len(notes) > 0 {
```

Comment:
Can we simplify this by just using https://pkg.go.dev/os#WriteFile ?

Reply:
Great suggestion, thanks! Done in https://github.com/google/osv.dev/pull/1039/commits/f609b90eaca02da4e3d95f4406113c66f16ec01f